File: als-server/jvm/src/main/scala/org/mulesoft/als/server/lsp4j/extension/CleanDiagnosticTreeParams.java
package org.mulesoft.als.server.lsp4j.extension;


import org.eclipse.lsp4j.TextDocumentIdentifier;

public class CleanDiagnosticTreeParams {
    private TextDocumentIdentifier textDocument;

    public CleanDiagnosticTreeParams(TextDocumentIdentifier document) {
        this.textDocument = document;
    }

    public TextDocumentIdentifier getTextDocument() {
        return textDocument;
    }
}

File: als-server/shared/src/main/scala/org/mulesoft/als/server/feature/diagnostic/CleanDiagnosticTreeParams.scala
package org.mulesoft.als.server.feature.diagnostic

import org.mulesoft.lsp.feature.common.TextDocumentIdentifier

/** The clean validation provides the list of all the files depending of the given uri (tree) and the eventual list of
  * diagnostics for each one.
  *
  * @param textDocument
  */
case class CleanDiagnosticTreeParams(textDocument: TextDocumentIdentifier)
